### PR TITLE
Add Mish activation function

### PR DESCRIPTION
**Pull request by Boseong-Seo**
- Now, user can use Mish activation function like torch or tensorflow.

**Self evaluation**:
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: nntrainer/layers/acti_func.h [hunk 1]
```diff
@@ -78,15 +78,18 @@ class ActiFunc {
       in_place = false;
       this->setActivation<Tensor>(gelu<T>, geluPrime<T>);
       break;
+    case ActivationType::ACT_ELU:
```
Comment (myungjoo) [inline nntrainer/layers/acti_func.h:81]: Do you have any reason to move this code block? It is not recommended to add unrelated code changes in a commit. It makes harder to track changes and bugs in the future.

**Review by myungjoo — COMMENTED** [review]
LGTM except for the unrelated code move in a switch-case.

File: nntrainer/layers/acti_func.h (hunk 1 above)
Comment (Boseong-Seo) [reply nntrainer/layers/acti_func.h:81]: @myungjoo Just for the code readability and uniformity(?).
All activation types except for `ACT_ELU` were placed on top of `ACT_NONE` in the switch-case. So, I moved the block to that place for some kind of uniformity(I wonder this kind of thing can be also said as unifomity...). But if you think it doesn't need to, then I'll rollback this change.

**Review by DonghakPark — APPROVED** [review]
LGTM!

File: test/unittest/unittest_nntrainer_activations.cpp [hunk 2]
```diff
@@ -517,6 +517,59 @@ TEST(nntrainer_activation, eluPrime_01_p) {
...
+  int height = 1;
+  int width = 10;
+
+  float answer[30] = {
+    -0.18891649, -0.15113318, -0.10714479, -0.05678857, 0.00000000,
+    0.06317942,  0.13259898,  0.20800139,  0.28903052,  0.37524524,
+    -0.28396326, -0.24693604, -0.18891649, -0.10714479, 0.00000000,
+    0.13259898,  0.28903052,  0.46613652,  0.65969974,  0.86509836,
+    -0.30883580, -0.29565641, -0.24693604, -0.15113318, 0.00000000,
+    0.20800139,  0.46613652,  0.76120591,  1.07794595,  1.40337825};
```
Comment (djeong20) [inline test/unittest/unittest_nntrainer_activations.cpp:532]: quick question. how are you getting this answer data? PyTorch, TensorFlow, or hand calculation?

Comment (Boseong-Seo) [reply test/unittest/unittest_nntrainer_activations.cpp:532]: With PyTorch!
I got the answer data by using `torch.nn.Mish()`.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by EunjuYang — APPROVED** [review]
LGTM

**Review by skykongkong8 — APPROVED** [review]

**Review by djeong20 — APPROVED** [review]
LGTM

File: nntrainer/layers/acti_func.h (hunk 1 above)
Comment (myungjoo) [reply nntrainer/layers/acti_func.h:81]: Uh.. then never mind. It's good to go.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**